### PR TITLE
Extract columns page

### PR DESCRIPTION
**Pull request by JeffDotPng**
Added page for `EXTRACT_COLUMNS` node

### CODE REVIEW

File: docs/nodes/AI_ML/CLASSIFICATION/ACCURACY/ACCURACY.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import AppendixSection from '@site/src/components/AppendixSection';
+
+[//]: # 'TODO: Machine-generate this section'
+
+import DocstringSource from '!!raw-loader!./a1-[autogen]/docstring.txt';
+import PythonSource from '!!raw-loader!./a1-[autogen]/python_code.txt';
+import ParametersSource from '!!raw-loader!./a1-[autogen]/parameters.yaml';
+
+<DocString>{DocstringSource}</DocString>
+<PythonCode GLink='AI_ML/CLASSIFICATION/SUPPORT_VECTOR_MACHINE.py'>{PythonSource}</PythonCode>
```
Comment (smahmed776) [inline docs/nodes/AI_ML/CLASSIFICATION/ACCURACY/ACCURACY.md:17]: GLink should be path to current node's file in nodes repo.

File: docs/nodes/AI_ML/CLASSIFICATION/ACCURACY/ACCURACY.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Examples
+
+import Example1 from './examples/EX1/example.md';
+import App1 from '!!raw-loader!./examples/EX1/app.txt';
+import Data1 from '!!raw-loader!./examples/EX1/output.txt';
+
+<AppDisplay 
+    data={Data1}
+    nodeLabel='SUPPORT_VECTOR_MACHINE'>
```
Comment (smahmed776) [inline docs/nodes/AI_ML/CLASSIFICATION/ACCURACY/ACCURACY.md:32]: Node label seems incorrect here

File: docs/nodes/AI_ML/CLASSIFICATION/ACCURACY/ACCURACY.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+[//]: # 'Appendix'
+
+import Notes from './appendix/notes.md';
+import Hardware from './appendix/hardware.md';
+import Media from './appendix/media.md';
+
+## Appendix
+
+<AppendixSection index={0} folderPath='nodes/AI_ML/CLASSIFICATION/appendix/'>{Notes}</AppendixSection>
```
Comment (smahmed776) [inline docs/nodes/AI_ML/CLASSIFICATION/ACCURACY/ACCURACY.md:49]: Folder path seems invalid for Accuracy node in all of AppendixSection component's folderPath prop.

File: docs/nodes/AI_ML/CLASSIFICATION/SUPPORT_VECTOR_MACHINE/SUPPORT_VECTOR_MACHINE.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import AppendixSection from '@site/src/components/AppendixSection';
+
+[//]: # 'TODO: Machine-generate this section'
+
+import DocstringSource from '!!raw-loader!./a1-[autogen]/docstring.txt';
+import PythonSource from '!!raw-loader!./a1-[autogen]/python_code.txt';
+import ParametersSource from '!!raw-loader!./a1-[autogen]/parameters.yaml';
+
+<DocString>{DocstringSource}</DocString>
+<PythonCode GLink='AI_ML/CLASSIFICATION/SUPPORT_VECTOR_MACHINE.py'>{PythonSource}</PythonCode>
```
Comment (smahmed776) [inline docs/nodes/AI_ML/CLASSIFICATION/SUPPORT_VECTOR_MACHINE/SUPPORT_VECTOR_MACHINE.md:17]: Is this path valid? nodes path should be same across docs and nodes repo.

File: docs/nodes/AI_ML/CLASSIFICATION/SUPPORT_VECTOR_MACHINE/SUPPORT_VECTOR_MACHINE.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+[//]: # 'Appendix'
+
+import Notes from './appendix/notes.md';
+import Hardware from './appendix/hardware.md';
+import Media from './appendix/media.md';
+
+## Appendix
+
+<AppendixSection index={0} folderPath='nodes/AI_ML/CLASSIFICATION/appendix/'>{Notes}</AppendixSection>
```
Comment (smahmed776) [inline docs/nodes/AI_ML/CLASSIFICATION/SUPPORT_VECTOR_MACHINE/SUPPORT_VECTOR_MACHINE.md:48]: Folder path seems invalid

File: docs/nodes/AI_ML/OBJECT_DETECTION/OBJECT_DETECTION/OBJECT_DETECTION.md
Comment (smahmed776) [inline docs/nodes/AI_ML/OBJECT_DETECTION/OBJECT_DETECTION/OBJECT_DETECTION.md:1]: If we're considering `object_detection` as a sub category under `AI_ML` the we can rename it to plural form IMO

File: docs/nodes/EXTRACTORS/DATAFRAME/EXTRACT_COLUMNS/EXTRACT_COLUMNS.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+## Examples
+
+import Example1 from './examples/EX1/example.md';
+import App1 from '!!raw-loader!./examples/EX1/app.txt';
+import Data1 from '!!raw-loader!./examples/EX1/output.txt';
+
+<AppDisplay 
+    data={Data1}
+    nodeLabel='SUPPORT_VECTOR_MACHINE'>
```
Comment (smahmed776) [inline docs/nodes/EXTRACTORS/DATAFRAME/EXTRACT_COLUMNS/EXTRACT_COLUMNS.md:32]: Again node label seems incorrect

File: docs/nodes/EXTRACTORS/DATAFRAME/EXTRACT_COLUMNS/a1-[autogen]/python_code.txt [hunk 7]
```diff
@@ -0,0 +1,66 @@
+from flojoy import flojoy, DataContainer
+import pandas as pd
+import numpy as np
+from sklearn import svm, preprocessing
+from typing import cast
+
+
+@flojoy
+def SUPPORT_VECTOR_MACHINE(
```
Comment (smahmed776) [inline docs/nodes/EXTRACTORS/DATAFRAME/EXTRACT_COLUMNS/a1-[autogen]/python_code.txt:9]: Incorrect python code for Extract columns

File: docs/nodes/EXTRACTORS/DATAFRAME/EXTRACT_COLUMNS/a1-[autogen]/parameters.yaml [hunk 8]
```diff
@@ -0,0 +1,11 @@
+target:
```
Comment (smahmed776) [inline docs/nodes/EXTRACTORS/DATAFRAME/EXTRACT_COLUMNS/a1-[autogen]/parameters.yaml:1]: Are they correct for Extract columns node?

File: docs/nodes/EXTRACTORS/DATAFRAME/EXTRACT_COLUMNS/examples/EX1/app.txt [hunk 9]
```diff
@@ -0,0 +1,166 @@
+{
```
Comment (smahmed776) [inline docs/nodes/EXTRACTORS/DATAFRAME/EXTRACT_COLUMNS/examples/EX1/app.txt:1]: I think Extract columns node isn't included in app

File: docs/nodes/EXTRACTORS/DATAFRAME/EXTRACT_COLUMNS/examples/EX1/example.md [hunk 10]
```diff
@@ -0,0 +1,3 @@
+In this example, the `SUPPORT_VECTOR_MACHINE` is passed the [iris dataset](https://archive.ics.uci.edu/dataset/53/iris), split into two parts. The training data contains 120 labels examples, while the input dataset contains 30 samples with the labels removed.
```
Comment (smahmed776) [inline docs/nodes/EXTRACTORS/DATAFRAME/EXTRACT_COLUMNS/examples/EX1/example.md:1]: Incorrect example md file?

**Review by smahmed776 — CHANGES_REQUESTED** [review]

File: docs/nodes/AI_ML/OBJECT_DETECTION/OBJECT_DETECTION/OBJECT_DETECTION.md
Comment (JeffDotPng) [reply docs/nodes/AI_ML/OBJECT_DETECTION/OBJECT_DETECTION/OBJECT_DETECTION.md:1]: A plural form how?

**Pull request merged**